recipes/git.rb

```diff
@@ -30,3 +30,14 @@
     to "#{node['drush']['install_dir']}/drush"
   end
 end
+
+# We have to run composer against the directory if we using 6.x or above
+require_recipe "composer"
+
+execute 'install-drush-deps' do
+  command "composer require"
```

Comment:
Wrong composer command. Also makes assumptions about the location an name of the composer command. Also you need to exclude dev dependencies. Take a look at how I handle this.